packages/next/build/webpack/plugins/telemetry-plugin.ts

```diff
@@ -137,11 +142,21 @@ export class TelemetryPlugin implements webpack.WebpackPluginInstance {
         callback()
       }
     )
+    compiler.hooks.compilation.tap(TelemetryPlugin.name, (compilation) => {
+      const moduleHooks = NormalModule.getCompilationHooks(compilation)
+      moduleHooks.loader.tap(TelemetryPlugin.name, (loaderContext: any) => {
+        loaderContext.eliminatedPackages = ELIMINATED_PACKAGES
```

Comment:
This approach won't work with caching. Packages would only be reported when they are compiled the first time.

Reply:
And in watch mode you would report outdated packages.

Reply:
Yeah that's ok - getting anything back here is valuable even if its not 100% accurate